knative/client.go

```diff
@@ -49,6 +50,16 @@ func NewEventingClient(namespace string) (clienteventingv1beta1.KnEventingClient
 	return client, nil
 }
 
+func NewKubernetesClientset(namespace string) (*kubernetes.Clientset, error) {
```

Comment:
This function is not referenced anywhere else in the PR. Is it intended? Maybe I'm missing something.

Reply:
It is being used in the function that checks whether Secret is present in the cluster:
https://github.com/boson-project/func/pull/369/files#diff-3bc17fd6241d546478336fa19a33f7e7f4c232421ce4e8511512c00fa617860dR490